aya/src/util.rs

```diff
@@ -364,6 +364,18 @@ pub(crate) fn bytes_of_slice<T: Pod>(val: &[T]) -> &[u8] {
     unsafe { slice::from_raw_parts(val.as_ptr().cast(), size) }
 }
 
+pub(crate) fn bytes_of_bpf_name(bpf_name: &[i8]) -> &[u8] {
```

Comment:
This won't be i8 for all architectures unfortunately (I've run into this before)

I think you'd either need to make use of generics OR just duplicate the logic for now unless @tamird Knows of a rusty way around this :)

Reply:
Or maybe use a ffi c type?

Reply:
Done. Works well with the ffi c type.